tests/unit_tests/test_wave_06.py

```diff
@@ -261,7 +281,18 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result is False
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f in jesse.inventory
+    assert item_a not in jesse.inventory
+    assert item_b not in jesse.inventory
+    assert item_c not in jesse.inventory
+    assert item_d not in tai.inventory
+    assert item_e not in tai.inventory
+    assert item_f not in tai.inventory
```

Comment:
Same as before, is there a way where we wouldn't have to check each individual item's location in a specific list?